PhotoGether/PresentationLayer/PhotoRoomFeature/PhotoRoomFeature/Source/ViewController/PhotoRoomViewController.swift

```diff
@@ -13,13 +26,78 @@ public class PhotoRoomViewController: BaseViewController, ViewControllerConfigur
     
     public override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .blue
+        view.backgroundColor = .black
+        
+        addViews()
+        setupConstraints()
+        configureUI()
+        bindInput()
+        bindOutput()
     }
     
-    public func addViews() { }
+    public func addViews() {
+        self.addChild(participantsViewController)
+        
+        [navigationView, participantsViewController.view, photoRoomBottomView].forEach {
+            view.addSubview($0)
+        }
+        
+        participantsViewController.didMove(toParent: self)
+    }
+    
+    public func setupConstraints() {
+        navigationView.snp.makeConstraints {
+            $0.top.equalTo(view.safeAreaLayoutGuide)
+            $0.horizontalEdges.equalToSuperview()
+            $0.height.equalTo(Constants.navigationHeight)
+        }
+        
+        participantsViewController.view.snp.makeConstraints {
+            $0.top.equalTo(navigationView.snp.bottom)
+            $0.horizontalEdges.equalToSuperview()
+            $0.bottom.equalTo(photoRoomBottomView.snp.top)
+        }
+        
+        photoRoomBottomView.snp.makeConstraints {
+            $0.bottom.equalTo(view.safeAreaLayoutGuide)
+            $0.horizontalEdges.equalToSuperview()
+            $0.height.equalTo(Constants.bottomViewHeight)
+        }
+    }
     
-    public func setupConstraints() { }
+    public func configureUI() {
+        navigationView.backgroundColor = PTGColor.gray50.color
+    }
     
-    public func configureUI() { }
+    public func bindInput() {
+        // MARK: Host에 필요한 Input만 guard문 아래에
+        guard isHost else { return }
+        
+        photoRoomBottomView.cameraButtonTapped
+            .sink { [weak self] _ in
+                self?.input.send(.cameraButtonTapped)
+            }
+            .store(in: &cancellables)
```

Comment:
```suggestion        
        photoRoomBottomView.cameraButtonTapped
            .filter {  isHost }
            .sink { [weak self] _ in
                self?.input.send(.cameraButtonTapped)
            }
            .store(in: &cancellables)
```

bindInput에서 host가 아니더라도 바인딩이 필요한 이벤트가 존재할 수도 있을 것 같아요.
filter를 통해서 host일때만 동작해야하는 이벤트에만 달아주는 건 어떨까요?

Reply:
분기를 단순히 '호스트일 때만' 과 '호스트, 게스트 양 측일때' 두 경우만 생각해서 저렇게 구현했습니다. 생각해보니 '게스트인 경우만'일 때는 기존 방식에서 처리하기 힘들 것 같네요! 
[973429f](https://github.com/boostcampwm-2024/iOS04-PhotoGether/pull/58/commits/973429f0ccc504188e10cf18a66de3a9e7f7b6f3) 수정했습니다!